src/main/java/ua/ithillel/lesson6/MyUtils.java

```diff
@@ -0,0 +1,106 @@
+package ua.ithillel.lesson6;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyUtils {
+
+    /*
+      1. Створити метод countOccurance, що приймає на вхід рядковий список як параметр і довільний рядок. Список
+      заповнений довільними словами 10-20 штук, які можуть повторюватись у необмеженій кількості. Порахувати скільки
+      разів зустрічається переданий рядок як другий аргумент.
+    */
+    public int countOccurrence(List<String> list, String item){
```

Comment:
```suggestion
    public static int countOccurrence(List<String> list, String item){
```
для інформації: в цьому класі доречно static на всіх методах використовувати.

Reply:
Спасибо, исправил